src/collections/_documentation/workflow/integrations/integration-platform/index.md

```diff
@@ -3,166 +3,573 @@ title: Integration Platform
 
-The examples below demonstrate a potential use-case that involves a Flask app receiving new issue webhooks from Sentry, calling the Sentry API for more data about the issue, and pushing it to [Pushover](https://pushover.net/) as a generator of desktop/mobile notifications. 
+### Permissions
 
-### Creating an Integration
+Permissions specify what level of access your service requires of Sentry resources. For public integrations, Sentry will prompt users to approve of these permissions upon installation. For more information, see the [full documentation on Permissions]({%- link _documentation/api/permissions.md -%}).
 
-In Sentry, navigate to Organization Settings > Developer Settings > Create New Integration.
+[{% asset integration-platform-index/permissions.png alt="Form that allows developer to set what permissions they'll need from their user." %}]({% asset integration-platform-index/permissions.png @path %})
 
-[{% asset integration-platform-index/create-new-integration.png alt="Form that allows user to fill in their integration details." %}]({% asset integration-platform-index/create-new-integration.png @path %})
+### Integration Webhooks
+Webhooks allows your service to get requests about specific resources, depending on your selection. For more information, see the [full documentation on Webhooks](#webhooks).
 
-### Permissions
+## Public Integrations
 
-Permissions specify what level of access your service requires of Sentry resources. Sentry will prompt your user to approve of these permissions upon installation. For more information on Permissions, [see full documentation]({%- link _documentation/api/permissions.md -%}).
+Sentry built public integrations for the 'general public' of Sentry users. Public integrations start in an unpublished state for development purposes and can later be submitted for approval to publish. For more information, see the [section on Publishing](#published-state).
 
-[{% asset integration-platform-index/permissions.png alt="Form that allows developer to set what permissions they'll need from their user." %}]({% asset integration-platform-index/permissions.png @path %})
+The  code examples in the sections below demonstrate a potential use-case that involves a Flask app receiving new issue webhooks from Sentry, calling the Sentry API for more data about the issue, and pushing it to Pushover as a generator of desktop/mobile notifications.
+
+###  Installation
 
-### Resource Subscriptions
+Users will have the option to install your integrations on the Integrations Page in Sentry. If your integration is still in an unpublished state, the only Sentry organization that will be able to install that integration will be the organization that created the integration. Clicking 'Install' will allow users to see a description of your integration and the option to 'Continue' accepting permissions needed for installation. 
```

Comment:
The 'continue' part is actually not going to be the case since I'm redo-ing the modals in https://github.com/getsentry/sentry/pull/14417 so 
```
...and the option to 'Continue' accepting permissions needed for installation. 
```
should probs become 
```
and the permissions that will be granted should the user choose to install. 
```

something like that